packages/plugin-auto-tracking-browser/src/libs/finder.ts

```diff
@@ -0,0 +1,322 @@
+/* istanbul ignore file */
+
+// License: MIT
+// Author: Anton Medvedev <anton@medv.io>
+// Source: https://github.com/antonmedv/finder
+
+type Knot = {
```

Comment:
nit: would it make sense to fork this repo and import it? That way we can pull in changes from the main repo in the future for bug fixes etc.

Reply:
Good question, I should have mentioned more in the PR desc. Reasons why I copy to folk here:
- I was trying to use that package from npm directly, but the package install works. It fails when building with the code, as it contains TS source file, seems to be an https://github.com/rollup/plugins/issues/287 of rollup, and the code also has TS errors unless manually fixing minor types to pass in our build. Not sure whether there is a good solution on this, but I cannot figure it out after investigating for a day.
- If we folk to our repo, as we don't have a private registry, we probably need to publish to npm to installation either, which will likely introduce the similar issue above and we need to manage a new package.
- The original repo is not updated very often, and we will probably add our own logic when tuning the selector results.

I think we can definitely folk to a new repo (so we can sync from main repo easier) and manage it in the future if needed. It should be easily replaceable here.

Reply:
BTW I think I fixed this issue upstream (https://github.com/antonmedv/finder/pull/83). I'm hoping once the new build is published it will fix the compilation issue.